intellij-settings/flows-config-schema.json

```diff
@@ -43,14 +33,30 @@
                 }
               }
             },
-            "subflow": {
+            "onPostAction": {
               "type": "string",
-              "description": "For screen within a subflow, they need the name of the subflow defined."
+              "description": "A method name to run on a POST."
+            },
+            "crossFieldValidationAction": {
+              "type": "string",
+              "description": "A method name to to validate one field based on another field."
```

Comment:
"An Action that will be called after general validation is done on the fields, but before the data is saved to the database.  It is a chance to do any cross field validation necessary."

Reply:
I'd also fill the rest out some more as well.  If this is the only documentation that a SNE sees, then it's good if we give a little more context.

Reply:
Done!